docs/release-notes/2.20.0.md

```diff
@@ -0,0 +1,78 @@
+# Percona Monitoring and Management 2.20.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 3, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- The [Easy-install script] for PMM Server is checksum verified.
+- PMM will use [Grafana 7.5] instead of 7.3. We are also preparing for Grafana 8.
+- PostgreSQL monitoring with the `pg_stat_monitor` plugin enabled exposes new metrics such as Plan Times, WAL Fpi/Bytes/Records.
+- For users who deploy PMM Server through the AWS Marketplace, AWS RDS service discovery will be executed without AWS credentials and tuning [IAM roles].
+- For Backup Management (Technical Preview), we added the ability to schedule backups so you can schedule and see already scheduled [backups] in the UI.
+
+## Important note for users of PMM 2.16.0 via Docker
+
+If you started using PMM from version 2.16 and have already upgraded to 2.17, 2.18, or 2.19, then you need to perform additional steps after upgrading to 2.20. (This only applies to Docker-based installations.)
```

Comment:
Isn't this for https://jira.percona.com/browse/PMM-8492
if yes - we need to remove this ad the issue is fixed, as I can see

@denisok ^ Am I correct about the assumption it's fixed?

Reply:
https://jira.percona.com/browse/PMM-8446

Reply:
that is new bug

Reply:
oops, my link is wrong, you are right it is 8492. And issue is fixed only for 2.21 for all other versions - either update to 2.21 required or manual steps.

Reply:
@nikita-b  `or still use 2,16` ? should we say that if you also use 2.16 this action is required ?

Reply:
@denisok Yes. you are right. It's also impacted 2.16 without updates

Reply:
To be clear:
Customers need manual action only if they started to use 2.16 and already updated to the version prior 2.20. If they didn't have an update we'll fix it during the update procedure